code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm

```diff
@@ -20,7 +21,18 @@
 /mob/camera/eminence/Move(NewLoc, direct)
 	var/OldLoc = loc
 	if(NewLoc && !istype(NewLoc, /turf/open/indestructible/reebe_void))
-		forceMove(get_turf(NewLoc))
+		var/turf/T = get_turf(NewLoc)
+		if(T.flags_1 & NOJAUNT_1)
+			if(last_failed_turf != T)
+				T.visible_message("<span class='warning'>[T] suddenly emits a ringing sound!</span>", ignore_mob = src)
+				playsound(T, 'sound/machines/clockcult/ark_damage.ogg', 75, FALSE)
+				last_failed_turf = T
+			to_chat(src, "<span class='warning'>This turf is consecrated and can't be crossed!</span>")
+			return
+		if(istype(get_area(T), /area/chapel))
+			to_chat(src, "<span class='warning'>The Chapel is hallowed ground under a heretical deity, and can't be accessed!</span>")
+			return
```

Comment:
When I finish my blessing changes, the chapel will just start with blessing objects in its area.

Reply:
Oh, I see.